client/my-sites/customer-home/components/home-survey/index.jsx

```diff
@@ -0,0 +1,50 @@
+import { useIsEnglishLocale } from '@automattic/i18n-utils';
+import ReactDOM from 'react-dom';
+import SurveyModal from 'calypso/components/survey-modal';
+import { useSelector } from 'calypso/state';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+import homeSurveyImage from './assets/images/home-survey.png';
+
+const HomeSurvey = () => {
+	const site = useSelector( getSelectedSite );
+	const isEnglishLocale = useIsEnglishLocale();
+	const isSiteLaunched = site?.launch_status === 'launched' || false;
+	const isSiteCreatedInLast3Days =
+		site?.options?.created_at &&
+		new Date( site.options.created_at ) > new Date( Date.now() - 3 * 24 * 60 * 60 * 1000 );
+	const isLastActivityWithin2Days =
+		site?.options?.updated_at &&
+		new Date( site.options.updated_at ) > new Date( Date.now() - 2 * 24 * 60 * 60 * 1000 );
```

Comment:
We can extract these numeric values as a variable to avoid magic numbers.

I'm also thinking if we have some date helpers available. We can use `moment` too.